coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -1837,47 +1837,67 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
     validatePlan(execPlan, expectedPlan)
   }
 
-  it("Modifier should have PeriodicSamplesMapper wrapped by RepeatTransformer") {
+  it("Modifier should have PeriodicSamplesMapper wrapped by RepeatTransformer and all data in raw cluster") {
```

Comment:
Could you add a test case for each possible scenario?
- query routed entirely to raw (already exists)
- query routed entirely to DS (already exists)
- query routed to both (should reject)
- query start/end are in raw, but lookback extends into DS (should reject)
- query start/end are in raw, but offset pushes it entirely into DS (ok)
- [any other important scenarios I'm forgetting]